### ISSUE TITLE
[Feature Request] Kindly add Translation

### ISSUE THREAD

**Issue by param1903** [TDesktop-x64/tdesktop#58]
### Is your feature request related to a problem?

Hi,
Thank you for the app but if possible kindly add the translation feature / option.

### Describe the solution you'd like

no solution

### Describe alternatives you've considered

No alternative

**Comment by matifali** [comment]
Yes. After getting used to translation in the android app, mow it is a must needed feature for desktop.
